packages/apollo-server-core/package.json

```diff
@@ -41,7 +41,7 @@
     "graphql-extensions": "file:../graphql-extensions",
     "graphql-tag": "^2.9.2",
     "graphql-tools": "^4.0.0",
-    "graphql-upload": "^8.0.2",
+    "graphql-upload": "^10.0.0",
```

Comment:
Update this to version 11.0.0.
One of the changes in version 11.0.0 is that graphql-upload is actaually CI tested for node version 14. 
Version 10.0.0 of graphql-upload is not CI tested for node version 14. See release notes here: https://github.com/jaydenseric/graphql-upload/releases